go-controller/Makefile

```diff
@@ -23,6 +23,7 @@ CONTAINER_RUNTIME=docker
 endif
 CONTAINER_RUNNABLE ?= $(shell $(CONTAINER_RUNTIME) -v > /dev/null 2>&1; echo $$?)
 OVN_SCHEMA_VERSION ?= v24.03.1
+OVS_VERSION ?= v3.2.0
```

Comment:
Let's discuss what this means for us since we don't have control of OVS versions deployed on hosts. 

We would not be able to support older server schema versions
https://github.com/ovn-org/libovsdb/issues/235
https://github.com/ovn-org/libovsdb/pull/257

We would be able to support newer server schema versions:
https://github.com/ovn-org/libovsdb/blob/ce1951614a200fcc854a3165b880a3fffdfbcb9b/mapper/mapper.go#L60

So this basically sets the minimum version we would support.

Reply:
I have set this to 2.17 per our discussions.